csharp/Ice/Filesystem/Server/MDirectory.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) ZeroC, Inc.
+
+namespace Server;
+
+/// <summary>Provides an in-memory implementation of the Slice interface Directory.</summary>
+internal class MDirectory : Filesystem.DirectoryDisp_
+{
+    private readonly string _name;
+    private readonly List<Filesystem.NodePrx> _contents = new();
+
+    /// <inheritdoc/>
```

Comment:
There is currently nothing to inherit currently as slice2cs neglects to generate doc-comments for server-side interfaces / classes.